app/models/relationship.rb

```diff
@@ -270,7 +269,11 @@ def propagate_inversion
         claim.project_media_id = self.source_id
         claim.save
       end
-      Relationship.where(source_id: self.target_id).update_all({ source_id: self.source_id })
+      Relationship.where(source_id: self.target_id).find_each do |r|
+        r.source_id = self.source_id
+        r.skip_check_ability = true
+        r.save!
+      end
```

Comment:
@caiosba I replaced `update_all` with save to run callbacks as I see the unmatched & parent_id not updated

Reply:
Thanks @melsawy, good catch.